src/lib/datadog-importer/index.ts

```diff
@@ -30,14 +31,18 @@ export const DatadogImporter = (
       console.log('Global Config is required');
       return inputs;
     }
-    const instanceIdTag = globalConfig['instance-id-tag'];
-    const locationTag = globalConfig['location-tag'];
-    const instanceTypeTag = globalConfig['instance-type-tag'];
-    const cpuUtilizationMetricName =
-      globalConfig['cpu-utilization-metric-name'];
+    const instanceIdTag: string = globalConfig['instance-id-tag'];
+    const locationTag: string = globalConfig['location-tag'];
+    const instanceTypeTag: string = globalConfig['instance-type-tag'];
+    const metric: string = globalConfig['metric'];
```

Comment:
Eventually, I'd like to support multiple metrics. That can be a follow up ticket if we don't want to do it now.

Reply:
Added support for multiple metrics so the output looks like this. Let me know if this is what you were thinking!
<img width="432" alt="Screenshot 2024-06-20 at 10 36 06 AM" src="https://github.com/moin-oss/datadog-importer/assets/11634794/99a3e3d9-397d-4ceb-91fc-a15a5fed64ec">

Reply:
LGTM